src/super_gradients/training/utils/callbacks/callbacks.py

```diff
@@ -856,6 +856,22 @@ def _infer_global_step(self, context: PhaseContext, is_train_loader: bool):
             return total_steps_in_done + train_loader_length + context.batch_idx
 
 
+@register_callback(Callbacks.SLIDING_WINDOW_INFERENCE)
```

Comment:
Why choose an inconsistent name? This just adds more confusion....

Reply:
Fixed on c79d0f2